.github/workflows/codeql-analysis.yml

```diff
@@ -32,7 +32,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        language: [ 'javascript' ]
+        language: [ 'cpp' ]
```

Comment:
Should we keep both cpp and javascript for this project? we do have both types of files present

Reply:
.js file is just a test which we (try to) ignore in the code ql config. Although I see now that the default config looks for a "tests" folder and we have a "test" folder. Fixing.